python/lsst/ts/ess/controller/command_handler.py

```diff
@@ -115,6 +115,7 @@ def create_device(
                 name=device_configuration[common.Key.NAME],
                 device_id=device_configuration[common.Key.FTDI_ID],
                 sensor=sensor,
+                baud_rate=device_configuration[common.Key.BAUD_RATE],
```

Comment:
You might consider reducing duplication by making a dict of the keywords common to all devices:
```
common_device_kwargs = dict(
    name=device_configuration[common.Key.NAME],
    sensor=sensor,
    callback_func=self._callback,
    log=self.log,
)
...
    device = VcpFtdi(
        device_id=device_configuration[common.Key.FTDI_ID],
        baud_rate=device_configuration[common.Key.BAUD_RATE],
        **common_device_kwargs,
    )
```